app/assets/javascripts/api_stub.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Created by kaceyryan on 5/2/15.
+ */
+
+var request = require('request');
+
+class API {
+  //get images in JSON object for your location
+  getImages(callback) {
```

Comment:
Correction: they should accept an `errorCallback` if they actually do anything with it ;)